dracut/parse-kickstart

```diff
@@ -521,7 +521,10 @@ def write_ifcfg(filename, ifcfg):
 
 def process_kickstart(ksfile):
     handler = DracutHandler()
-    handler.ksdevice = os.environ.get('ksdevice')
+    try:
+        handler.ksdevice = proc_cmdline['ksdevice']
+    except KeyError:
+        log.info("ksdevice argument is not available")
```

Comment:
Ok true. 
I think it's good to have this debug message but I'll change it to debug instead of info.

I will change it locally.